### DISCUSSION TITLE
https://docs.datalust.co/docs/serilogtracing is missing a nuget package

### DISCUSSION THREAD

**Discussion by ghost** [datalust/seq-tickets#2097] [Help]
You have to also include the `SerilogTracing.Expressions` nuget package for the following line to compile in the example given in the docs on the noted page:
`.WriteTo.Console(Formatters.CreateConsoleTextFormatter(TemplateTheme.Code))`

**Comment by liammclennan** [comment]
@wes-mytrailmate Thank you. That has been updated.
